packages/typespec-ts/src/modular/helpers/operationHelpers.ts

```diff
@@ -437,10 +445,30 @@ function getCollectionFormat(param: Parameter) {
   }
   const isMulti = (param.format ?? "").toLowerCase() === "multi";
   const additionalParam = isMulti ? `, "${param.restApiName}"` : "";
-  if (!param.optional) {
-    return `"${param.restApiName}": ${collectionInfo}(${param.clientName}${additionalParam})`;
+  const serializeHelperImport = importSet.get("../rest/index.js");
+  if (serializeHelperImport) {
+    serializeHelperImport.add(collectionInfo);
+  } else {
+    importSet.set("../rest/index.js", new Set<string>().add(collectionInfo));
```

Comment:
I don't think we need to add the importSet here because:
- the default fix missing import could help us
- the fixed path `../rest/index.js` may not work in multi client.

Reply:
will remove this, previously it's not working because we fail to export it in the rest/index.js.

Reply:
resolved.